src/platform/Ameba/Logging.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <platform_stdlib.h>
+
+#define AMEBA_LOGLEVEL_ERROR 1
```

Comment:
how about using C++ constants or better yet, enums?

Reply:
Fixed, thank you